screego.config.example

```diff
@@ -36,6 +37,13 @@ SCREEGO_TURN_PORT_RANGE=
 # remote ip.
 SCREEGO_TURN_STRICT_AUTH=true
 
+# If true, screego will not start TURN server and instead use an external one
+# hosted on the address you specify via SCREEGO_EXTERNAL_IP.
+SCREEGO_TURN_EXTERNAL=false
+
+# Authentication secret for the external TURN server.
+SCREEGO_TURN_EXTERNAL_SECRET=
+
```

Comment:
I feel like the settings are a little scattered. SCREEGO_EXTERNAL_IP is used for the turn server ip, but SCREEGO_TURN_ADDRESS is used for the port of the external turn ip.

I think these should be separate settings, what do you think of configuration like this:
```
# If set, screego will not start TURN server and instead use a external TURN server
# Example: 192.168.178.2:3456
SCREEGO_TURN_EXTERNAL_ADDRESS=

# Authentication secret for the external TURN server.
SCREEGO_TURN_EXTERNAL_SECRET=
```

Reply:
I agree! Am I right that we should then change `SCREEGO_EXTERNAL_IP` to not be required, instead one of `SCREEGO_EXTERNAL_IP` or `SCREEGO_TURN_EXTERNAL_ADDRESS` must be required?

Reply:
Yes, we somehow have to make this clear in the documentation otherwise it probably won't be used (:.

Reply:
Because we must also support dual IP stack, I went the similar route as you did for internal TURN - created `SCREEGO_TURN_EXTERNAL_IP` which matches `SCREEGO_EXTERNAL_IP` and is parsed in exact same way (reusing your parsing logic), and `SCREEGO_TURN_EXTERNAL_PORT` (defaulting to the same port `3478`).

I guess this makes it even easier for people to reason about, that they need either `SCREEGO_EXTERNAL_IP` or `SCREEGO_EXTERNAL_TURN_IP`.

What do you think of this?